export.py

```diff
@@ -92,6 +158,19 @@ def json_to_markdown(json):
 {body}
 """.format(**json)
 
+def get_cookie_value(response, cName):
+    try:
+        header = response.headers.get('Set-Cookie')
+
+        if header:
+            return header.split(f'{cName}=')[1].split(';')[0]
+        else:
+            raise ValueError(f'Cookie {cName} not found in response.')
+
+    except Exception as e:
+        print(f"Error extracting cookie: {cName}. Error: {e}. Exiting...")
+        sys.exit(1)
```

Comment:
```suggestion
        sysexit(1)
```
I think this is a typo

Reply:
Thank you!